airbyte-server/build.gradle

```diff
@@ -37,9 +39,10 @@ dependencies {
     implementation 'org.glassfish.jersey.inject:jersey-hk2'
     implementation 'org.glassfish.jersey.media:jersey-media-json-jackson'
     implementation 'org.glassfish.jersey.ext:jersey-bean-validation'
+    implementation 'io.opentelemetry:opentelemetry-api'
+    implementation 'io.opentelemetry:opentelemetry-extension-annotations'
```

Comment:
Per our discussion, we don't need the OpenTelemetry dependencies in airbyte-server.